src/accordion/accordion-item.component.ts

```diff
@@ -38,6 +38,7 @@ import {
 export class AccordionItem {
 	static accordionItemCount = 0;
 	@Input() title: string | TemplateRef<any>;
+	@Input() context?: Object;
```

Comment:
I don't think this needs to be made into an optional input, or maybe we can have it something like `@Input() context: Object | null = null` since it seems that it takes both type Object or null https://angular.io/api/common/NgTemplateOutlet

Reply:
I changed it, but I was thinking... If this input is not optional and the user doesn't write it, wouldn't it be `undefined` instead of `null`?